quadlets/cfe18.container

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CFE_18 container
+Requires=cfe18mariadb.service
+
+[Container]
+Image=ghcr.io/teragrep/cfe_18:2.1.1
```

Comment:
Maybe could use `:latest` instead of `:2.1.1` as version so this file stays automatically up to date